ts/features/walletV3/onboarding/components/WalletOnboardingError.tsx

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable functional/immutable-data */
+import * as React from "react";
+import * as O from "fp-ts/lib/Option";
+import { pipe } from "fp-ts/lib/function";
+
+import { OnboardingOutcome } from "../types";
+import GenericErrorComponent from "../../../fci/components/GenericErrorComponent";
```

Comment:
I don't know if this is temporary or not, but `GenericErrorComponent ` is a specific component exported from FCI feature, which implements specific code for FCI use-cases. This is not good.
I'd suggest to replace it with a custom component for Wallet specific use-cases.
In other words: I'd suggest to make a `GenericErrorComponent` for the Wallet feature and place it inside`ts/features/walletV3/common/components` 😄

Reply:
This is a temporary screen meanwhile a final design will be ready. Anyway I created an independent `GenericErrorComponent` inside `ts/feature/walletV3/common/components` folder as you suggested ([2d50633](https://github.com/pagopa/io-app/pull/4810/commits/2d5063304f55de1653688acedb5a5b1d3beeabc3)) 💪.

Of course the UI is momentary until Design provides us a Figma, in that case as soon as it will be available we can address the implementation in another PR.